src/client/app/chat/chat.component.ts

```diff
@@ -548,15 +588,25 @@ export class ChatComponent implements OnInit, AfterViewChecked {
   //getDoctors
   getDoctors() {
     if (this.doctorList) {
+     // this.safeUrl=this.domSanitizer.bypassSecurityTrustResourceUrl(this.doctor.videoUrl);
       this.chatService.getDoctors(this.userId)
         .subscribe((doctors) => {
           doctors.map((doctor: any) => {
             this.doctors.push(doctor);
             this.ref.detectChanges();
           });
         });
+       
     }
     this.doctorList = false;
+  }
+   //open Video Modal
+  video(videoModal:any){
+  this.modalService.open(videoModal).result.then((result) => {
+    this.closeResult = `Closed with: ${result}`;
+  }, (reason) => {
+    this.closeResult = `Dismissed ${this.getDismissReason(reason)}`;
```

Comment:
Similar to the previous comment, we are not interested in knowing the cause of closing modal window. Kindly clean this code.